core/libp2p/host.hpp

```diff
@@ -49,15 +49,20 @@ namespace libp2p {
     /**
      * @brief Let Host handle all protocols with prefix={@param prefix}, for
      * which predicate {@param predicate} is true.
-     * param prefix prefix for the protocol. Example: /ping/
+     * param prefix prefix for the protocol. Example:
+     * You want all streams, which came to Ping protocol with versions
+     * between 1.5 and 2.0 to be handled by your handler; so, you may pass
+     *  - prefix=/ping/1.
+     *  - predicate=[](proto){return proto.version >= 1.5
```

Comment:
but proto here is a string?

Reply:
Yeah, this is just a simplified pseudo example